hack/crd-e2e.sh

```diff
@@ -46,7 +46,7 @@ trap cleanup INT TERM
 res=0
 
 # Install kind
-(go install sigs.k8s.io/kind@v0.17.0) || res=$?
+(go install sigs.k8s.io/kind@v0.22.0) || res=$?
 
```

Comment:
hey @astoycos : re code below:
```
for _ in bases patches; do
  go run sigs.k8s.io/controller-tools/cmd/controller-gen rbac:roleName=manager-role crd paths=./apis/... output:crd:dir=./config/crd/bases output:stdout || res=$?
  kubectl kustomize config/crd | kubectl apply -f - || res=$?
  # Temporary workaround for https://github.com/kubernetes/kubernetes/issues/104090
  sleep 8
done
```
do you know if we also try to add experimental CRDs after the standard here? I see in CI: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_network-policy-api/194/pull-network-policy-api-crd-e2e/1759934589674983424
```
customresourcedefinition.apiextensions.k8s.io/adminnetworkpolicies.policy.networking.k8s.io created
customresourcedefinition.apiextensions.k8s.io/baselineadminnetworkpolicies.policy.networking.k8s.io created
customresourcedefinition.apiextensions.k8s.io/adminnetworkpolicies.policy.networking.k8s.io configured
customresourcedefinition.apiextensions.k8s.io/baselineadminnetworkpolicies.policy.networking.k8s.io configured
Deleting cluster "verify-network-policy-api" ...
```
two sets for sure, just wanted to confirm if that second one is experimental or not.. wanna ensure we have coverage for both

Reply:
Hrmmm so in the kustomize file in `config/crd` looks like 

```
resources:
- standard/policy.networking.k8s.io_adminnetworkpolicies.yaml
- standard/policy.networking.k8s.io_baselineadminnetworkpolicies.yaml
```

Which would only install the standard CRDs, I think we should push the kustomization.yaml down into the standard/experimental directories and then apply/delete them explicitly

```
kubectl kustomize config/crd/standard | kubectl apply -f -
kubectl kustomize config/crd/standard | kubectl delete -f -
kubectl kustomize config/crd/experimental | kubectl apply -f -
kubectl kustomize config/crd/experimental | kubectl delete -f -
```

Reply:
ah good point true! So I guess the card I opened in issues was valid : https://github.com/kubernetes-sigs/network-policy-api/issues/192
Shall I fix this in a new PR which is smaller, this one as too much changes as is, I don't want to break stuff